iati/validator.py

```diff
@@ -228,6 +228,20 @@ def contains_warnings(self):
         'description': 'The XML prolog must occur at the start of the document.',
         'info': '{err}',
         'help': 'The XML prolog specifies how a computer must read the rest of the XML file. Since it tells the computer how to read the XML file, it must occur at the start of an XML document without any content before it.\nIt looks similar to: `<?xml version="1.0" encoding="UTF-8"?>`.\nFor more information about the XML prolog, see https://www.w3schools.com/xml/xml_syntax.asp'
+    },
+    'err-encoding-invalid': {
+        'base_exception': ValueError,
+        'category': 'file',
+        'description': 'The encoding specified within the XML prolog is different from the actual encoding of the XML file.',
```

Comment:
`The encoding specified within the XML prolog is different...` -> `The value specified within the XML encoding declaration is different...` ?

At least, 'encoding declaration' seems to be used here: https://www.w3.org/TR/2000/REC-xml-20001006#NT-EncodingDecl